engine/access/rpc/backend/script_executor_test.go

```diff
@@ -0,0 +1,287 @@
+package backend
+
+import (
+	"context"
```

Comment:
the linter wants all stdlib imports to be in a separate group. See the goland config I recommended in discord. it helps automate this.